engine/reference/commandline/dockerd.md

```diff
@@ -1090,71 +1092,71 @@ This is a full example of the allowed configuration options on Linux:
 
 ```json
 {
-	"authorization-plugins": [],
```

Comment:
They won't be merged back at all. Instead, the contents will be re-copied from `docker/docker` at some point in the future, at which point the newly added options would show up.